tests/ha/ha_cluster_init.pm

```diff
@@ -94,6 +88,17 @@ sub run {
     # If we failed to initialize the cluster with 'ha-cluster-init', trying again with crm in debug mode
     cluster_init('crm-debug-mode', $fencing_opt, $unicast_opt, $qdevice_opt) if (!wait_serial("ha-cluster-init-finished-0", $join_timeout));
 
+    # Set wait_for_all option to 0 if we are in a two nodes cluster situation
+    # We need to set it for reproducing the same behaviour we had with no-quorum-policy=ignore
+    if (!get_var('TWO_NODES=no')) {
```

Comment:
Not sure about this syntax for `get_var()`. Perhaps you need to change this to `!check_var('TWO_NODES', 'no')` or to `get_var('TWO_NODES', 'yes') !~ m/no/i`.

In fact, it's configuring `wait_for_all: 0` even in the 3 node cluster test: http://1a102.qa.suse.de/tests/5268#step/ha_cluster_init/16

Reply:
yes I did a mix between `get_var` and `check_var`...  it was too late last night :smile: